dockerHashing/src/main/scala/cromwell/docker/local/DockerCliClient.scala

```diff
@@ -33,7 +34,7 @@ trait DockerCliClient {
     * @return Failure if an error occurs.
     */
   def pull(dockerCliKey: DockerCliKey): Try[Unit] = {
-    forRun("docker", "pull", dockerCliKey.fullName) { _ => () }
+    forRun("docker", "pull", dockerCliKey.fullName) { const(()) }
   }
```

Comment:
I'd previously changed code around here and @mcovarr saw fit to comment on nearby code. I wound up removing my changes but left this in to prevent heartache for the next soul that modified something within a few lines of this